webui/.prettierrc

```diff
@@ -0,0 +1,13 @@
+{
+  "semi": true,
+  "trailingComma": "all",
+  "singleQuote": false,
```

Comment:
This might be risky.
How was this applied? Any chance it might've messed up some string containing a single quote in the middle?